submissions/trostinetska/hooli-style-popup/index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="author" content="Larysa Trostinetska" />
+    <meta name="description" content="popup" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Popup</title>
+    <link rel="stylesheet" href="styles.css" type="text/css" />
+  </head>
+  <body>
+    <header>
+      <nav class="navbar">
+        <ul class="crumbs">
+          <li class="crumb"><a href="#">Mail</a></li>
+          <li class="crumb"><a href="#">Images</a></li>
+          <li class="crumb">
+            <input
+              type="checkbox"
+              id="checkbox_toggle"
+              name="checkbox_toggle"
+              class="check_input"
+            />
+            <label for="checkbox_toggle" class="check">
+              <img src="./icons/popup-button.png" alt="start-menu" />
+            </label>
+            <nav class="popup">
+              <ul class="popup_list">
+                <li class="popup_item">
+                  <a class="popup_link" href="#"
+                    ><img
+                      class="icon"
+                      src="./icons/calendar.png"
+                      alt=""
+                    />Calendar</a
+                  >
+                </li>
+                <li class="popup_item">
+                  <a class="popup_link" href="#"
+                    ><img class="icon" src="./icons/mail.png" alt="" />Mail</a
+                  >
+                </li>
```

Comment:
Fix your styles on focus)
![image](https://user-images.githubusercontent.com/16196199/184417328-fc815e6e-14f6-455c-a3d3-8c437c12b13d.png)

Reply:
Thanks) I don't understand, what do I need to do? Should I delete focus-within and work only with focus? Or...?

Reply:
By default browser use `outline` to draw an outline around the focused element, and you use border on focus. You must choose what approach you will use.

Reply:
Thank you) I'll try to do this.